### PR TITLE
Qualimap - Rest of the plots

### PR DESCRIPTION
**Pull request by guillermo-carrasco**
Here the rest of the plots from QualiMap. Sorry, the relevant edited file is `multiqc/qualimap/__init__.py`, the rest is test data. The test data is the result of running qualimap on the data for [this](http://trace.ncbi.nlm.nih.gov/Traces/sra/?study=ERP000975) study.

Ready for your review!

### CODE REVIEW

File: multiqc/qualimap/__init__.py [hunk 1]
```diff
@@ -28,10 +30,16 @@ def __init__(self):
...
+
         # Find QualiMap reports
         qualimap_raw_data = {}
         for root, dirnames, filenames in os.walk(config.analysis_dir, followlinks=True):
-            if 'genome_results.txt' in filenames and 'raw_data_qualimapReport' in dirnames:
+            raw_data_dir = 'raw_data'
+            for d in dirnames:
+                if raw_data_dir in d:
+                    raw_data_dir = d
+            if 'genome_results.txt' in filenames and raw_data_dir in dirnames:
```
Comment (ewels) [inline multiqc/qualimap/__init__.py]: Why do you need to look for a directory at all? Why can't you just look for a file called `genome_results.txt`?

File: multiqc/qualimap/__init__.py [hunk 2]
```diff
@@ -28,10 +30,16 @@ def __init__(self):
...
+        self.parsed_stats = defaultdict(dict)
+
         # Find QualiMap reports
         qualimap_raw_data = {}
         for root, dirnames, filenames in os.walk(config.analysis_dir, followlinks=True):
-            if 'genome_results.txt' in filenames and 'raw_data_qualimapReport' in dirnames:
+            raw_data_dir = 'raw_data'
+            for d in dirnames:
+                if raw_data_dir in d:
+                    raw_data_dir = d
```
Comment (ewels) [inline multiqc/qualimap/__init__.py]: Does it actually matter what the directory is called? Can't you just look for a file called `genome_results.txt` and assume that whatever directory the `os.walk` is in at that time (`root`) is the data dir?

File: multiqc/qualimap/__init__.py [hunk 3]
```diff
@@ -40,12 +48,10 @@ def __init__(self):
...
                 if s_name in qualimap_raw_data:
                     log.debug("Duplicate sample name found! Overwriting: {}".format(s_name))
-                
+
                 qualimap_raw_data[s_name] = {}
-                qualimap_raw_data[s_name]['reports'] = {os.path.splitext(r)[0]: os.path.join(root, 'raw_data_qualimapReport', r) \
-                    for r in os.listdir(os.path.join(root, 'raw_data_qualimapReport'))}
-                qualimap_raw_data[s_name]['plots'] = {os.path.splitext(r)[0]: os.path.join(root, 'images_qualimapReport', r) \
-                    for r in os.listdir(os.path.join(root, 'images_qualimapReport'))}
+                qualimap_raw_data[s_name]['reports'] = {os.path.splitext(r)[0]: os.path.join(root, raw_data_dir, r) \
```
Comment (ewels) [inline multiqc/qualimap/__init__.py]: In fact, if you're looking for files with specific filenames (keys looked for below), can't you just look for files with those filenames?

**Comment by ewels** [comment]
Looks awesome! Thanks again for writing this..

**Comment by guillermo-carrasco** [comment]
Thank you for the assistance and teaching! :-D

**Pull request merged**